web/src/title/NewProjectMode.svelte

```diff
@@ -52,6 +52,8 @@
       return;
     }
 
+    // REIVEW: can we change to be like this to align with ltn_cnt project keys?
```

Comment:
This would affect newly created projects. But in `getProjectList()`, we still have the old format. We could look for both there, or decide to make a breaking change and switch to `ltn/`. I think we _should_ do that breaking cutover for consistency, but maybe not just yet? Related to #195 also.